root/release_group/merge.tt

```diff
@@ -22,7 +23,8 @@
                     <td>
                         <input type="hidden" name="merge.merging.[% loop.index %]" value="[% entity.id %]" />
                         <input type="radio" name="merge.target" value="[% entity.id %]"
-                            [% 'checked="checked"' IF entity.id == form.field('target').value %] />
+                            [% 'checked="checked"' IF entity.id == form.field('target').value %]
+                            [% 'disabled="disabled"' IF to_merge.size <= 1 %] />
```

Comment:
It would be nice to avoid showing a non actionable merge page with only one selected entity, either by redirecting to the still selected entity, or by only allowing to cancel the merge when two entities are selected for merge.

Reply:
Made that change.